iosApp/iosApp/Pages/NearbyTransit/NearbyTransitView.swift

```diff
@@ -33,6 +33,7 @@ struct NearbyTransitView: View {
     @State var pinnedRoutes: Set<String> = []
     @State var predictionsByStop: PredictionsByStopJoinResponse?
     var errorBannerRepository = RepositoryDI().errorBanner
+    let noNearbyStops: () -> NoNearbyStopsView
```

Comment:
question(non-blocking): Why take this as a function instead of a view?

Reply:
It seemed like the more idiomatic SwiftUI approach.